### PR TITLE
Update PPP

### PR DESCRIPTION
**Pull request by CharlesLee01**

### CODE REVIEW

File: docs/team/charleslee01.md [hunk 1]
```diff
@@ -11,28 +11,41 @@ command box provided.
 
 Given below are my contributions to the project.
 
-* **New Feature**: Edit Command
-  * Implemented the EditEntityCommand, EditOwnerCommand and EditDogCommand that edits owners and dogs in the database.
-  * The EditOwnerCommand and EditDogCommand class are implemented such that it extends from the EditEntityCommand.
-  * In the process, I had to build `EditEntityCommandParser`, `EditOwnerCommandParser` and `EditDogCommandParser`to allow the proper functioning of this command.
+* **Code Refactoring**: Code Quality Improvements
+  * Increase defensive coding, exceptions and assertions.
+  * Update Javadoc comments so that it fits the current context.
```
Comment (ZhangAnli) [inline docs/team/charleslee01.md:16]: Can you rephrase this portion, seems like you are patch-writing from my PPP. And make sure PPP reflects your unique contributions, and that you have commits to back up what you write here.

File: docs/team/charleslee01.md [hunk 2]
```diff
@@ -11,28 +11,41 @@ command box provided.
...
 * **Documentation**:
   * User Guide:
-    * Contributed significant portions in initial layout of the user guide. Navigating the User Guide,
-      Edit Command and Command Summary.
-    * Maintain the consistency of the user guide. Make sure each command format is accurate.
-    * Use icons such as :bulb: and :heavy_exclamation_mark: to notify readers about possible warnings and hints when using Pawbook.
+    * Contributed significant portions in initial layout of the user guide. Navigating the User Guide, Edit Command and Command Summary.
+    * Maintain the consistency of the user guide. For instance, ensure the command formats are accurate.
+    * Ensure that sufficient examples and explanations are given so that the exact features of each command are clear to users. 
+    * Use icons such as :bulb: to notify readers about possible warnings and :heavy_exclamation_mark: to show the hints when using Pawbook.
```
Comment (shaelynl) [inline docs/team/charleslee01.md:43]: the bulb is for hints and the heavy exclamation mark is for warnings

Comment (CharlesLee01) [reply docs/team/charleslee01.md:43]: Yea, accidentally switched it during rephrasing. Thx!

**Review by shaelynl — APPROVED** [review]
